scripts/fastsp_script/fsp_test.py

```diff
@@ -2,39 +2,98 @@
 import sys
 
 """
-###remember sys.argv - can create variables using indexes with sys.argv!  
-    
+###remember sys.argv - can create variables using indexes with sys.argv!      
 """
 
 dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
 slash = "/"
+fas = ".fasta"
 
-#print(sys.argv)
-
-outfile = output_fs.txt
-#will need to be .csv later 
+infile = open("output_fs.txt", "w+")
+infile_name = "output_fs.txt"
+outfile = open("parsed_fs_output.csv","w")
+header = "reference,estimation,datatype,sp_score,tc"+ "\n"
+com = ","
+outfile.write(header)
+space = " "
 
 aa_ls_dir = os.listdir(dir)
-
-for file in aa_ls_dir:
-    if file.endswith(".fasta"):
-        #print (file)
-        for file2 in aa_ls_dir:
-            if file2.endswith(".fasta"):
-                if file != file2:
-                    print(file,file2)
-                    #print(dir+slash+file + dir+slash+file2)
-                    
-                    fasta1 = dir+slash+file
-                    fasta2 = dir+slash+file2
-                    
+                                
+for i in range(50):
+    for j in range (i,50):
+        if i != j:
+        
+            file = aa_ls_dir[i]
+            file2 = aa_ls_dir[j] 
+            
+            fasta1 = dir+slash+file
+            fasta2 = dir+slash+file2
+            #print(fasta1,fasta2)
+            
+            if file.endswith(fas):
+                if file2.endswith(fas):
+                    #print(file,file2)
                     print("Running FastSP analysis on " + file + " and " + file2 + "\n")
                     cmd = "java -jar FastSP.jar -r " + fasta1 + " -e " + fasta2
-                    print(cmd)
+                    #print(cmd)
+            
+                    fincmd = cmd + " > " + infile_name
+                    #print(fincmd)  
+                    os.system(fincmd)
+                    
+                    ########### code below only works if its only one loop 
+                    
+                    if_lines = infile.readlines()
```

Comment:
In the end, you'll want to have this in a function - after calling FastSP, call a function to parse its output and return an output string to be included in the final file. It may help to start by writing a totally separate practice script that ONLY parses an example FastSP output, then incorporate into here!